### ISSUE TITLE
TSL level not reported (and used as a filter) in all_epitope.tsv/aggregated.tsv 

### ISSUE THREAD

**Issue by PierreLaplante** [griffithlab/pVACtools#1044]
### Installation Type

Docker

### pVACtools Version / Docker Image

4.0.6

### Python Version

3.11.0

### Operating System

CentOs 7 HPC

### Describe the bug

The transcript support level is not reported and used as filtering with input VEP annotated vcf containing TSL field.

### How to reproduce this bug

```Shell
VEP command used:
vep \
    --species mus_musculus --assembly GRCm38 --no_stats --buffer_size 5000 --sift b \
    --ccds --uniprot --hgvs --symbol --numbers --domains --gene_phenotype --terms SO \
    --canonical --protein --biotype --uniprot --tsl --variant_class \
    --shift_hgvs 1 --check_existing --total_length --allele_number \
    --no_escape --xref_refseq --failed 1 --vcf --flag_pick_allele --transcript_version \
    --pick_order canonical,tsl,biotype,rank,ccds,length \
    --dir $HOME/.vep/ \
    --cache --fasta 102_GRCm38/Mus_musculus.GRCm38.dna.primary_assembly.fa \
    --format vcf \
	--plugin Frameshift --plugin Wildtype \
	--dir_plugins $HOME/vep_plugins/ \
    --input_file input.vcf\
    --output_file input_vep.vcf \
    --offline --pubmed --fork 4 --regulatory --verbose \

pvacseq command used :

singularity exec \
	--mount type=bind,src=SOURCE,dst=DEST \
	pvac406.sif pvacseq run \
	input.vcf.gz \
	id \
	H-2-Kd,H-2-Dd,H-2-Ld \
	NetMHCpan \
	output/dir \
	--iedb-install-directory /opt/iedb \
	--net-chop-method cterm \
	--netmhc-stab \
	--run-reference-proteome-similarity --blastp-path ncbi/dir --blastp-db refseq_select_prot \
	-a sample_name \
	--phased-proximal-variants-vcf phased.vcf.gz \
	--tdna-vaf 0.05 \
	--trna-vaf 0.05 \
```

### Input files

input vep_vcf, TSL field is present and full: 
[input.vcf.gz](https://github.com/griffithlab/pVACtools/files/13861720/input.vcf.gz)

### Log output

No logs.

### Output files

output all_epitope.tsv/aggregated.tsv showing Not Supported:
[output.all_epitopes.aggregated.txt](https://github.com/griffithlab/pVACtools/files/13861790/output.all_epitopes.aggregated.txt)

(truncated for upload size)
[output.all_epitopes.txt](https://github.com/griffithlab/pVACtools/files/13861791/output.all_epitopes.txt)

**Comment by PierreLaplante** [comment]
Is it possible for you guys to add a command to add TSL to already existing .all_epitope.tsv by crossing with the corresponding vcf? So that already long ran analysis can be annotated without rerunning the whole prediction.
Thank you.

**Comment by susannasiebert** [comment]
I don't really see a utility for such a command in the long run after this fix goes live. My suggestion would be to start a second run from scratch but abort it after the initial TSV gets created. That tsv will contain an index column that can be used to associate its entries with those from your all_eptiopes.tsv (using the Index column in the all_epitopes.tsv file). You can then use that to fix the TSL entries using a custom script.

**Comment by PierreLaplante** [comment]
I see, that makes sense, thank you.

**Comment by susannasiebert** [comment]
This issue should be fixed in version 4.0.7. Please give it a try and let me know if you're still running into this problem.
